Mathlib/Data/Fin/Basic.lean

```diff
@@ -934,6 +964,46 @@ def castSuccEmb : Fin n ↪o Fin (n + 1) :=
 #align fin.cast_lt_cast_succ Fin.castLT_castSucc
 #align fin.cast_succ_lt_cast_succ_iff Fin.castSucc_lt_castSucc_iff
 
+@[simp]
+theorem castSucc_le_castSucc_iff : castSucc a ≤ castSucc b ↔ a ≤ b := Iff.rfl
+@[simp]
```

Comment:
```suggestion

@[simp]
```
and similarly for the next few simp lemmas

Reply:
To be clear, this is just a spacing thing?